slave/process-mailaliases-generic/bin/process-mailaliases_generic.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
 
-PROTOCOL_VERSION='3.0.0'
+PROTOCOL_VERSION='3.0.1'
```

Comment:
You shouldn't modify `PROTOCOL_VERSION` as you don't receive data in the new/modified format. You just modified logic, which decides destination file.